x-pack/plugins/observability_solution/inventory/server/routes/has_data/route.ts

```diff
@@ -16,18 +14,16 @@ export const hasDataRoute = createInventoryServerRoute({
       requiredPrivileges: ['inventory'],
     },
   },
-  handler: async ({ context, logger }) => {
-    const coreContext = await context.core;
-    const inventoryEsClient = createObservabilityEsClient({
-      client: coreContext.elasticsearch.client.asCurrentUser,
-      logger,
-      plugin: `@kbn/${INVENTORY_APP_ID}-plugin`,
-    });
+  handler: async ({ plugins, request }) => {
+    const entityManagerStart = await plugins.entityManager.start();
+    const entityManagerClient = await entityManagerStart.getScopedClient({ request });
 
-    return getHasData({
-      inventoryEsClient,
-      logger,
+    const { total } = await entityManagerClient.v2.countEntities({
+      start: moment().subtract(15, 'm').toISOString(),
```

Comment:
are the `start`, `end` params required? 

 if I start ingesting data for couples of days and for some reason I stop ingesting data for the last 15m, this will return false

Usually, `getHasData` is checking if there is ANY data.

Reply:
I just saw your response above, @rmyz. Could you please share where this decision is documented?